nanovdb/nanovdb/cmd/updateFiles.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
We probably need a similar file that runs on Windows.

Reply:
yes good idea but I have no idea how to do scripting with string replacements in Windows